### ISSUE TITLE
Change xtrlock

### ISSUE THREAD

**Issue by BenjaminSchubert** [PolyProg/livesystem#12]
xtrlock seems a very weak protection. Should we remove it ?

**Comment by Sjlver** [comment]
It's weak... but unlocking all screens at the beginning gives a very clear signal to users when they can start the contest.

**Comment by BenjaminSchubert** [comment]
Yes sure. I was thinking about using another screenlocker that would not just be an overlay :)
